utils/CI/schema_validation.sh

```diff
@@ -33,6 +33,7 @@ validate() {
 
 validate_core() { validate "$1" ./wesnoth --validate data/_main.cfg; }
 validate_misc() { validate "$1" ./wesnoth --data-dir=. --validate=data/_main.cfg --preprocess-defines="$2"; }
+validate_achievements() { validate "Achievements schema" ./wesnoth --data-dir=. --validate=data/achievements.cfg; }
```

Comment:
This is validating the achievements, not the achievements schema, right?

Reply:
Yep, fixed.